core/test/junit/saros/concurrent/jupiter/test/puzzles/SimpleClientServerTest.java

```diff
@@ -33,21 +33,21 @@ public void setupClientServer(String initialText) {
    * the document states.
    */
   @Test
-  public void test2WayProtocol() throws Exception {
+  public void test2WayProtocol() {
 
     setupClientServer("abc");
 
-    client.sendOperation(new InsertOperation(0, "e"), 100);
+    client.sendOperation(I(0, "e"), 100);
```

Comment:
Because I only created the test helper class while migrating the tests, so I didn't want to go back and introduce it with the old logic and re-base the other changes on it.

But I can see how much work it takes to move it to a separate commit before the new position handling if you would prefer.

Reply:
Done.